### ISSUE TITLE
Weird training loss curve on VOC2007

### ISSUE THREAD

**Issue by yaoqi-zd** [endernewton/tf-faster-rcnn#187]
I followed the instructions in the README to train faster rcnn(vgg16 backbone) model on Pascal VOC 2007. My environment is Ubuntu14.04 + tensorflow r1.2 + GTX1080ti. After training, I get such total loss curve on tensorboard
![2](http://ofdz11dcz.bkt.clouddn.com/window_008.png)
the orange one is the train loss and the blue one is the val loss(on VOC 2007 test set), does these two loss curve seem normal? I'm a little confused

**Comment by endernewton** [comment]
quite normal to me.. did you get similar results after training?

**Comment by yaoqi-zd** [comment]
yes, I get ~70 mAP on VOC2007 test set(trained on VOC 2007 trainval). I just wonder why the training loss is oscillating and the performance on validation set doesn't get better as time goes by?

**Comment by yaoqi-zd** [comment]
@endernewton yes, I get ~70 mAP on VOC2007 test set(trained on VOC 2007 trainval). I just wonder why the training loss is oscillating and the performance on validation set doesn't get better as time goes by?

**Comment by endernewton** [comment]
according to traditional ML it is called overfitting, but maybe convnet has other intriguing properties except this one. closing.

**Comment by swg209** [comment]
@yaoqi-zd  Hi, I am quite curious about  how to make  two curves show together on a  graph? 

and  did you train your model at trainval set? Do you mean  training the model with the tfrecord create with trainval.txt in pascal_voc2007?   Thanks.
